src/pages/transactions/messages.tsx

```diff
@@ -5,6 +5,32 @@ import PaginatedMsgs from '@/components/safe-messages/PaginatedMsgs'
 import TxHeader from '@/components/transactions/TxHeader'
 import NavTabs from '@/components/common/NavTabs'
 import { transactionNavItems } from '@/components/sidebar/SidebarNavigation/config'
+import { Box, SvgIcon, Typography } from '@mui/material'
+import InfoIcon from '@/public/images/notifications/info.svg'
+import ExternalLink from '@/components/common/ExternalLink'
+import { useAppSelector } from '@/store'
+import { selectSafeMessages } from '@/store/safeMessagesSlice'
+
+const SignedMessagesHelpLink = () => {
+  const safeMessages = useAppSelector(selectSafeMessages)
+  const safeMessagesCount = safeMessages.data?.results.length ?? 0
+
+  if (safeMessagesCount === 0) {
+    return null
+  }
+
+  return (
+    <Box display="flex" alignItems="center" gap={1}>
+      <SvgIcon component={InfoIcon} inheritViewBox color="border" fontSize="small" />
+      {/* TODO: Add link to help article */}
+      <ExternalLink suppressIcon href="#">
```

Comment:
Could you do me a favor and replace all instances of `suppressIcon` with `noIcon`?

Reply:
sure! in a different PR I assume?

Reply:
Right, to dev.

Reply:
https://github.com/safe-global/web-core/pull/1460